modules/porous_flow/src/materials/PorousFlowPermeabilityUnity.C

```diff
@@ -0,0 +1,32 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "PorousFlowPermeabilityUnity.h"
+
+template<>
+InputParameters validParams<PorousFlowPermeabilityUnity>()
+{
+  InputParameters params = validParams<Material>();
+  params.addClassDescription("This Material calculates the permeability tensor assuming it is equal to 1 0 0  0 1 0  0 0 1");
+  return params;
+}
+
+PorousFlowPermeabilityUnity::PorousFlowPermeabilityUnity(const InputParameters & parameters) :
+    DerivativeMaterialInterface<Material>(parameters),
+    _PorousFlow_name_UO(getUserObject<PorousFlowDictator>("PorousFlowDictator_UO")),
+    _num_var(_PorousFlow_name_UO.numVariables()),
+    _permeability_qp(declareProperty<RealTensorValue>("PorousFlow_permeability_qp")),
```

Comment:
Thanks for calling this "PorousFlow_permeability_qp" - better to have the "_qp" to indicate it's at the quadpoints